### PR TITLE
Issue 1443 yang model

### LINKED ISSUE #1443: Some fixes for the Yang model
- Define the equation directly in terms of c_sei instead of L_sei (just a parameter change)
- Check parameter values
- Check why the capacity reported in the paper is 12.4 Ah

### PR DESCRIPTION
**Pull request by tinosulzer**
# Description

Fixes some parameters in the Yang model

Fixes #1443 

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [x] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [x] No style issues: `$ flake8`
- [ ] All tests pass: `$ python run-tests.py --unit`
- [ ] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

**Comment by tinosulzer** [comment]
Can't request a review from @KennethNwanoro for some reason

File: pybamm/input/parameters/lithium-ion/seis/yang2017_sei/README.md [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
 
-> Ramadass, P., Haran, B., Gomadam, P. M., White, R., & Popov, B. N. (2004). Development of first principles capacity fade model for Li-ion cells. Journal of the Electrochemical Society, 151(2), A196-A203.
-> Ploehn, H. J., Ramadass, P., & White, R. E. (2004). Solvent diffusion model for aging of lithium-ion battery cells. Journal of The Electrochemical Society, 151(3), A456-A462.
-> Single, F., Latz, A., & Horstmann, B. (2018). Identifying the mechanism of continued growth of the solid–electrolyte interphase. ChemSusChem, 11(12), 1950-1955.
-> Safari, M., Morcrette, M., Teyssot, A., & Delacour, C. (2009). Multimodal Physics-Based Aging Model for Life Prediction of Li-Ion Batteries. Journal of The Electrochemical Society, 156(3), 
 > Yang, X., Leng, Y., Zhang, G., Ge, S., Wang, C. (2017). Modeling of lithium plating induced aging of lithium-ion batteries: Transition from linear to nonlinear aging. Journal of Power Sources, 360, 28-40.
 
-Note: this parameter set does not claim to be representative of the true parameter values. Instead these are parameter values that were used to fit SEI models to observed experimental data in the referenced papers.
+and references therein
+
```
Comment (brosaplanella) [inline pybamm/input/parameters/lithium-ion/seis/yang2017_sei/README.md:8]: Remove blank line (will fix Codacy)

File: pybamm/input/parameters/lithium-ion/separators/separator_Yang2017/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Separator parameters
+
+Parameters for the separator from the papers
+
+> Ecker, Madeleine, et al. "Parameterization of a physico-chemical model of a lithium-ion battery i. determination of parameters." Journal of the Electrochemical Society 162.9 (2015): A1836-A1848.
+
+>Ecker, Madeleine, et al. "Parameterization of a physico-chemical model of a lithium-ion battery II. Model validation." Journal of The Electrochemical Society 162.9 (2015): A1849-A1857.
```
Comment (brosaplanella) [inline pybamm/input/parameters/lithium-ion/separators/separator_Yang2017/README.md:7]: You need a blank space (will fix Codacy)

**Review by brosaplanella — APPROVED** [review]
Looks good, thanks Tino!

**Comment by KennethNwanoro** [comment]
Looks good, I will try the model again when it's merged.
Regards.

**Pull request merged**